app/views/welcome/index.html.erb

```diff
@@ -5,6 +5,9 @@
 <% end %>
 <div class= "rectangle">
     <i class="fa fa-bars" style="fwidth: 20.3px; height: 13.5px; background-color: #000000;background-color: var(--black);"></i>
+    <% flash.each do |key, value| %>
```

Comment:
In the future, you can move this logic into a partial, it seems to be pretty generic.